network-api/networkapi/templates/donate/pages/help_page.html

```diff
@@ -22,7 +22,7 @@ <h1 class="tw-h1-heading"> {{ page.title }} </h1>
                 {% endblocktrans %}
             </p>
             <div class="tw-border-t tw-border-b tw-py-10 tw-mt-28 tw-mb-10 tw-border-gray-20">
-                <p class="tw-h3-heading">{% trans "Contact Us" %}</p>
+                <p id="contact-us" class="tw-h3-heading">{% trans "Contact Us" %}</p>
```

Comment:
Per request of @Miaatmozilla, we are now adding an ID so we can link to the "Contact Us" Section.